sdk/communication/communication-network-traversal/test.env

```diff
@@ -0,0 +1,7 @@
+# Used in most samples. Retrieve these values from a Communication Services instance
+# in the Azure Portal.
+COMMUNICATION_CONNECTION_STRING="endpoint=https://<resource name>.communication.azure.net/;accessKey=<key>"
```

Comment:
I believe we've recently changed this in the pipeline from COMMUNICATION_CONNECTION_STRING
to either COMMUNICATION_LIVETEST_STATIC_CONNECTION_STRING or COMMUNICATION_LIVETEST_DYNAMIC_CONNECTION_STRING. If this is similar to identity, COMMUNICATION_LIVETEST_DYNAMIC_CONNECTION_STRING should suffice